database/migrations/2020_10_05_162247_create_stores_table.php

```diff
@@ -16,10 +16,10 @@ public function up()
         Schema::create('stores', function (Blueprint $table) {
             $table->id();
             $table->text('name');
-            $table->longtext('description');
-            $table->foreignId('address_id')->constrained();
-            $table->text('url');
-            $table->text('image');
+            $table->longtext('description')->nullable();
+            $table->foreignId('address_id')->nullable();
```

Comment:
You still need to keep the `->constrained()` at the end!
`$table->foreignId('address_id')->nullable()->constrained();`